packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -392,15 +456,12 @@ const TagSubforumPage2 = ({classes}: {
         <Typography variant="display3" className={classes.title}>
           {tag.name}
         </Typography>
-        {!tag.wikiOnly && !editing && userHasNewTagSubscriptions(currentUser) && (
-          <SubscribeButton
-            tag={tag}
-            className={classes.notifyMeButton}
-            subscribeMessage="Subscribe"
-            unsubscribeMessage="Unsubscribe"
-            subscriptionType={subscriptionTypes.newTagPosts}
-          />
-        )}
+        {/* TODO change what appears in SubforumNotificationSettings list */}
+        {/* Join/Leave button always appears in members list, so only show join button here as an extra nudge if they are not a member */}
+        {!!currentUser && !editing && (isSubscribed ? <SubforumNotificationSettings tag={tag} currentUser={currentUser} className={classes.notificationSettings} /> : <SubforumSubscribeSection tag={tag} className={classes.joinBtn} />)}
```

Comment:
I ran into a bug where if I joined from the "Wiki" tab, then switch to the "Posts" tab, the "Members" list doesn't list me (until I refresh). It does work properly if I click that same button from the "Posts" tab. Prob not a blocker, feel free to record and address later.

Reply:
Hmm I can't reproduce this. I'll make a note to look into it though